app/src/main/java/com/example/lapnguyen/weather_3/screen/mainscreen/MainPresenter.java

```diff
@@ -0,0 +1,60 @@
+package com.example.lapnguyen.weather_3.screen.mainscreen;
+
+import android.app.Activity;
+import android.content.Context;
+import android.icu.text.SimpleDateFormat;
+import android.location.Location;
+import android.location.LocationListener;
+import android.location.LocationManager;
+import android.os.Bundle;
+import android.util.Log;
+import com.example.lapnguyen.weather_3.api.GPSTracker;
+import com.example.lapnguyen.weather_3.data.model.Weather;
+import com.example.lapnguyen.weather_3.data.source.MainRepository;
+import io.reactivex.android.schedulers.AndroidSchedulers;
+import io.reactivex.annotations.NonNull;
+import io.reactivex.disposables.CompositeDisposable;
+import io.reactivex.disposables.Disposable;
+import io.reactivex.functions.Consumer;
+import io.reactivex.schedulers.Schedulers;
+import java.util.Date;
```

Comment:
ERROR: Unused import: java.util.Date. Use Ctrl+Shift+O (Eclipse) or Ctrl+Alt+O (Intellij) to sort imports